recipe/meta.yaml

```diff
@@ -19,6 +19,7 @@ build:
     - mkdir -p $PREFIX/include                                                # [unix]
     - cp -r include/* $PREFIX/include/                                        # [unix]
     - mkdir -p $PREFIX/lib                                                    # [unix]
+    - mv lib/{{ cuda_major }}/*.so* $PREFIX/lib/                              # [unix]
```

Comment:
What does this line do?

Reply:
It's meant to make available `libcuda.so` to the tester for linking as in `cutensor`, [here](https://github.com/conda-forge/cutensor-feedstock/blob/main/recipe/meta.yaml#L34). It's missing this:
```
{% if cuda_compiler_version in (None, "None", True, False) %}
{% set cuda_major = 0 %}
{% else %}
{% set cuda_major = environ.get("cuda_compiler_version", "11.2").split(".")[0] | int %}
{% endif %}
```

Reply:
No, that's not what that line in cutensor recipe was doing. There it was for copying the cuTENSOR DSOS to the package location. But here, it's already done in the line below. We must not redistribute libcuda, it's a violation of CUDA EULA. If you need libcuda stub, the way to go is to add cuda-driver-dev to the test dependency, we did this in several feedstocks.

Reply:
> No, that's not what that line in cutensor recipe was doing. There it was for copying the cuTENSOR DSOS to the package location. But here, it's already done in the line below. We must not redistribute libcuda, it's a violation of CUDA EULA. If you need libcuda stub, the way to go is to add cuda-driver-dev to the test dependency, we did this in several feedstocks.

Thanks for this clarification. I misunderstood the logic as it was presented in the other feedstock. Indeed, I am very glad to be wrong as I was concerned with implications otherwise.

I have
1.  reverted the commit referencing this incorrect logic,
2.  added the appropriate test requirement, and 
3.  kept the changes regarding driver linking.

Let me know if this resolves the issue.